README.md

```diff
@@ -21,3 +21,22 @@ We welcome contributions from all members of the IT Society. Please follow these
 3.  Make your changes
 4.  Submit a pull request
 
+## Set up
+
+First run
+`pip install -r requirements.txt`
+
+then
+
+A env file is needed. make a `.env` file in the main directory of the project add add `SECRET_KEY = <key here>`
+if there are no users a admin account is needed. to do this add ADMIN_EMAIL = <email> and ADMIN_PASSWORD = <password> to the `.env` file.
+(these are needed for the first time the project is run please create a superuser account from the admin panel)
+
+Demo ENV file for first time setup
```

Comment:
Have you heard of .env.example? I think this would be a good thing to use here (as well as this)

Also, perhaps adding a link to both python and flask's documentation might be a good idea here

Reply:
why the python docs